front/app/components/admin/PostManager/components/LazyPostPreview/Idea/AdminIdeaEdit.tsx

```diff
@@ -279,9 +275,8 @@ class AdminIdeaEdit extends PureComponent<Props, State> {
     });
 
     this.setState({ processing: true, submitError: false });
-
     try {
-      if (oldImageId && oldImageBase64 !== newImageBase64) {
+      if (oldImageId) {
```

Comment:
Looking at this with fresh eyes, does it mean every time there is an old image, it will be deleted, regardless of whether the field has changed or not? Maybe we can set a state for this in the component `ideaImageChanged` that is false and the two callbacks - `onImageFileAdd` and `onImageFileRemove` set it to true? Similar to `ideaFilesChanged`.

Reply:
Sure I can do that! :+1: